docker-compose.open-solver.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.6"
+services:
+  stablex:
+    command: ./stablex
+    build:
+      context: .
+      dockerfile: docker/rust/Dockerfile
+      args:
+        - RUST_BASE=rust-binary
+    image: stablex-binary-public
+    environment:
+      - SOLVER_TYPE=open-solver
+      - RUST_BACKTRACE=1
```

Comment:
Can we not keep the parts that are common between the private and the open solver inside docker-compose.binary and add two configs that only specify the image and SOVLER_TYPE env variable? This way we would be duplicating less code.